featuretools/primitives/options_utils.py

```diff
@@ -58,9 +58,9 @@ def generate_all_primitive_options(all_primitives,
     for primitive in all_primitives:
         if primitive in primitive_options and primitive.name in primitive_options:
             msg = "Options present for primitive instance and generic " \
-                  "primitive class (%s), primitive instance will not use generic " \
+                  "primitive class %s, primitive instance will not use generic " \
```

Comment:
This one was really hard to work around, using `()` in a string seems to break something with `pytest.warn`. Tried using string literals but no luck.

Would like to know if someone knows how to work around this issue.

Reply:
I don't think it is critical to have the parentheses here, but I'm also not sure how to work around this exactly either.

Reply:
using escapes on the parentheses around mean --  "\\(%s\\)" -- in the `warning_msg` string should work.  The string is interpreted as a regex so a regular parentheses won't get the match we want